### PR TITLE
rclone: update to 1.58.1, install completions

### PR DESCRIPTION
**Pull request by daniel-eys**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

File: srcpkgs/rclone/template [hunk 1]
```diff
@@ -35,4 +36,9 @@ post_install() {
 	ln -sf rclone ${DESTDIR}/usr/bin/mount.rclone
 	vlicense COPYING
 	vman rclone.1
+
+	for shell in bash zsh fish; do
+		vtargetrun $GOPATH/bin/rclone genautocomplete $shell completion.$shell
```
Comment (4ricci) [inline srcpkgs/rclone/template:41]:
```suggestion
		vtargetrun ${DESTDIR}/usr/bin/rclone genautocomplete $shell completion.$shell
```

**Comment by anddam** [comment]
Any blocker on this?

File: srcpkgs/rclone/template (hunk 1 above)
Comment (daniel-eys) [reply srcpkgs/rclone/template]: Added your suggestion, thanks!

File: srcpkgs/rclone/template [hunk 2]
```diff
@@ -35,4 +36,9 @@ post_install() {
 	ln -sf rclone ${DESTDIR}/usr/bin/mount.rclone
```
Comment (4ricci) [inline srcpkgs/rclone/template:36]: Just one more thing:

```suggestion
	ln -s rclone ${DESTDIR}/usr/bin/mount.rclone
```

https://github.com/void-linux/void-packages/pull/33884#discussion_r744033333

Comment (daniel-eys) [reply srcpkgs/rclone/template]: done, thanks

Comment (paper42) [reply srcpkgs/rclone/template]: -sf is actually correct, we sometimes want to run one step twice and without -f, it would fail

**Comment by daniel-eys** [comment]
closing, superseded by #37991

**Pull request closed without merging**